packages/console/src/components/RoleUsersTransfer/components/SourceUserItem/index.tsx

```diff
@@ -0,0 +1,44 @@
+import type { User } from '@logto/schemas';
+import { useTranslation } from 'react-i18next';
+
+import Checkbox from '@/components/Checkbox';
+import UserAvatar from '@/components/UserAvatar';
+import { onKeyDownHandler } from '@/utilities/a11y';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  user: User;
+  isSelected: boolean;
+  onSelect: () => void;
+};
+
+const SourceUserItem = ({ user: { avatar, name }, isSelected, onSelect }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  return (
+    <div
+      role="button"
+      tabIndex={0}
+      className={styles.item}
+      onKeyDown={onKeyDownHandler(() => {
+        onSelect();
+      })}
+      onClick={() => {
+        onSelect();
+      }}
+    >
+      <Checkbox
+        checked={isSelected}
+        disabled={false}
```

Comment:
hmmm, I think `disabled` should be optional prop in `checkbox` component. Would you like to update in this PR?

Reply:
Maybe do it later to avoid endless rebase actions.
LOG-5176: Make the `disabled` prop optional in the `CheckBox` component